care/emr/api/viewsets/scheduling/booking.py

```diff
@@ -42,15 +47,22 @@ class RescheduleBookingSpec(BaseModel):
 
 
 class TokenBookingFilters(FilterSet):
-    status = CharFilter(field_name="status")
+    status = MultipleChoiceFilter(
```

Comment:
Lets keep it as a string filter, check other implementations around the codebase.